server/routes/v1/tokenDiscovery.js

```diff
@@ -114,21 +123,45 @@ async function cacheTokensList() {
   }
 }
 
-if (!timerStarted && process.env.REACT_APP_ENVIRONMENT === 'mainnet') {
-  timerStarted = true;
-  cacheTokensList();
-  setInterval(() => cacheTokensList(), TIME_INTERVAL);
+function startCaching() {
+  if (process.env.REACT_APP_ENVIRONMENT === 'mainnet') {
+    cacheTokensList();
+    const intervalId = setInterval(() => cacheTokensList(), TIME_INTERVAL);
+    if (!IS_PROD_ENV) {
+      setTimeout(() => {
+        log.info('stopping caching tokens');
+        clearInterval(intervalId);
+      }, TIME_TO_TEST);
```

Comment:
Why do we want the test environment to wait for an hour here? (Is it simulating how long it would take to actually get the tokens, or something else?)